Libraries/CMSIS/Device/Maxim/MAX32572/Include/fcr_regs.h

```diff
@@ -83,7 +83,7 @@ typedef struct {
     __IO uint32_t urvctrl;              /**< <tt>\b 0x14:</tt> FCR URVCTRL Register */
     __R  uint32_t rsv_0x18;
     __IO uint32_t gp;                   /**< <tt>\b 0x1C:</tt> FCR GP Register */
-    __IO uint32_t trimctrl;             /**< <tt>\b 0x20:</tt> FCR TRIMCTRL Register */
+    __IO uint32_t msradc9;              /**< <tt>\b 0x20:</tt> FCR MSRADC9 Register */
```

Comment:
@sihyung-maxim FYI register change - may need to update SVD/run the SVD workflow on  the PR into main.